adafruit_apds9960/apds9960.py

```diff
@@ -195,7 +201,7 @@ def __init__(
             self._write8(_APDS9960_GCONF4, 0)
             self._write8(_APDS9960_GPULSE, 0)
             self._write8(_APDS9960_ATIME, 255)
-            self._write8(_APDS9960_CONTROL, 1)
+            self._write8(_APDS9960_CONTROL, 0)
```

Comment:
can you revert this or explain it please!

Reply:
There are two reasons for this change: first, the code claims to set registers to power-on defaults, but according to the datasheet the power-on default for register CONTROL is 0x00. Bits 1:0 of this register are the AGAIN setting, and the existing code (setter-method color_gain) also documents that the power-on default value for AGAIN is 0.

The driver defaults for AGAIN is 1 and is set a few lines down.

Reply:
yep, datasheet for ref:
![image](https://user-images.githubusercontent.com/8755041/157263251-6f457c55-8d65-46e3-babc-5fe0bd4a4680.png)